src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationFeedbackPresenter.cs

```diff
@@ -165,6 +178,67 @@ private void updateFeedback(ParameterIdentificationRunState state)
          _activeFeedbackPresenter?.UpdateFeedback(state);
       }
 
+      public void OnFormClosed()
+      {
+         //Nothing to do here
+      }
+
+      public void Close()
+      {
+         closeAndClearReferences();
+      }
+
+      public IPresentationSettings GetSettings()
+      {
+         //Nothing to do here
+         return null;
+      }
+
+      public void RestoreSettings(IPresentationSettings settings)
+      {
+         //Nothing to do here
+      }
+
+      public void SaveChanges()
+      {
+         //Nothing to do here
+      }
+
+      public void Activated()
+      {
+         //Nothing to do here
+      }
+
+      public void Handle(RenamedEvent eventToHandle)
+      {
+         //Nothing to do here
+      }
+
+      public void Edit(ParameterIdentificationFeedback objectToEdit)
+      {
+         _parameterIdentificationFeedback = objectToEdit;
+
+         switch (objectToEdit.RunStatus)
+         {
+            case RunStatusId.Running:
+               setParameterIdentificationToStarted(objectToEdit.ParameterIdentification);
+               break;
+            case RunStatusId.Canceled:
+               setParameterIdentificationToStarted(objectToEdit.ParameterIdentification);
```

Comment:
This is weird. Why start and then cancel. I am sure there is a reason but I would comment this in the code